code/diagram/solutions/de.itemis.mps.editor.diagram.runtime/models/de/itemis/mps/editor/diagram/runtime/jgraph.mps

```diff
@@ -135375,5 +134917,5320 @@
+            <node concept="3uibUv" id="5IQEFjDEw3e" role="1tU5fm">
+              <ref role="3uigEE" node="190K99KhFiz" resolve="MyGraph" />
+            </node>
+            <node concept="2OqwBi" id="5IQEFjDEw3C" role="33vP2m">
+              <node concept="37vLTw" id="5IQEFjDEw3D" role="2Oq$k0">
+                <ref role="3cqZAo" node="5IQEFjDEuER" resolve="activeDiagram" />
+              </node>
+              <node concept="liA8E" id="5IQEFjDEw3E" role="2OqNvi">
+                <ref role="37wK5l" node="4HMzb$XPdI$" resolve="getContextGraph" />
+              </node>
+            </node>
+          </node>
+        </node>
+        <node concept="3clFbF" id="YGA9S4UmDW" role="3cqZAp">
+          <node concept="1rXfSq" id="YGA9S4UmDU" role="3clFbG">
+            <ref role="37wK5l" node="YGA9S4Ry00" resolve="fitSizeAll" />
+            <node concept="37vLTw" id="YGA9S4UmPu" role="37wK5m">
+              <ref role="3cqZAo" node="5IQEFjDEw3B" resolve="graph" />
+            </node>
+            <node concept="37vLTw" id="YGA9S4WLYl" role="37wK5m">
+              <ref role="3cqZAo" node="YGA9S4WJQ_" resolve="selectionOnly" />
+            </node>
+          </node>
+        </node>
+      </node>
+      <node concept="3Tm1VV" id="YGA9S4RJtk" role="1B3o_S" />
+      <node concept="3cqZAl" id="YGA9S4RKlN" role="3clF45" />
+    </node>
+    <node concept="2tJIrI" id="YGA9S4RImR" role="jymVt" />
+    <node concept="2YIFZL" id="YGA9S4Ry00" role="jymVt">
+      <property role="TrG5h" value="fitSizeAll" />
+      <node concept="3clFbS" id="YGA9S4Ry03" role="3clF47">
+        <node concept="3cpWs8" id="YGA9S4RAnt" role="3cqZAp">
+          <node concept="3cpWsn" id="YGA9S4RAnu" role="3cpWs9">
+            <property role="TrG5h" value="model" />
+            <node concept="3uibUv" id="YGA9S4RAnv" role="1tU5fm">
+              <ref role="3uigEE" node="1JWQbhaGymU" resolve="MyGraphModel" />
+            </node>
+            <node concept="2OqwBi" id="YGA9S4RAnw" role="33vP2m">
+              <node concept="liA8E" id="YGA9S4RAny" role="2OqNvi">
+                <ref role="37wK5l" node="4E9wbMRWh_b" resolve="getModel" />
+              </node>
+              <node concept="37vLTw" id="YGA9S4RBoI" role="2Oq$k0">
+                <ref role="3cqZAo" node="YGA9S4Ryhf" resolve="graph" />
+              </node>
+            </node>
+          </node>
+        </node>
+        <node concept="3clFbF" id="YGA9S4RAnz" role="3cqZAp">
+          <node concept="2OqwBi" id="YGA9S4RAn$" role="3clFbG">
+            <node concept="37vLTw" id="YGA9S4RAn_" role="2Oq$k0">
+              <ref role="3cqZAo" node="YGA9S4RAnu" resolve="model" />
+            </node>
+            <node concept="liA8E" id="YGA9S4RAnA" role="2OqNvi">
+              <ref role="37wK5l" to="99ht:~mxGraphModel.beginUpdate()" resolve="beginUpdate" />
+            </node>
+          </node>
+        </node>
+        <node concept="3cpWs8" id="YGA9S4WPr9" role="3cqZAp">
```

Comment:
All statements after `.beginUpdate()` in `DiagramActionsUtil.fitSizeAll` should be moved into the `try` block. Otherwise, it's not guaranteed that `.endUpdate()` is called if one of them throws an exception.